Changelog.md

```diff
@@ -1,5 +1,73 @@
 ## Gazebo Physics 6.x
 
+### Gazebo Physics 6.6.0 (2024-06-11)
+
+1. dartsim: optimize picking contact points with ODE collision detectorq
+    * [Pull request #584](https://github.com/gazebosim/gz-physics/pull/584)
+
+1. Fix windows compiler warning
+    * [Pull request #629](https://github.com/gazebosim/gz-physics/pull/629)
+
+1. Disable test failing due to ODE/libccd
+    * [Pull request #621](https://github.com/gazebosim/gz-physics/pull/621)
+
+1. bullet-featherstone: Improve mesh collision stability
+    * [Pull request #600](https://github.com/gazebosim/gz-physics/pull/600)
+
+1. bullet-featherstone: Support nested models
+    * [Pull request #574](https://github.com/gazebosim/gz-physics/pull/574)
+
+1. Revert "bazel: updates for garden (#513)"
+    * [Pull request #513)"](https://github.com/gazebosim/gz-physics/pull/513)")
+
+1. bazel: updates for garden
+    * [Pull request #513](https://github.com/gazebosim/gz-physics/pull/513)
```

Comment:
This can be removed

Reply:
Removed in [804a3ef](https://github.com/gazebosim/gz-physics/pull/652/commits/804a3ef4532f1b0000686ea8f0aca56b3f9780de)